cli/status/lua/instance_state.lua

```diff
@@ -0,0 +1,57 @@
+local ok, config = pcall(require, 'config')
+if ok and config ~= nil and type(config) == "table" and type(config.info) == "function" then
```

Comment:
This `config.info` type check has non-obvious reason and I would leave a remark. We can explicitly mention that `moonlibs/config` has the same module name and that it shouldn't be considered as tarantool 3.x config module.

Reply:
Maybe check the version Tarantool's major version here too?

Reply:
> Maybe check the version Tarantool's major version here too?

The existing check seems enough, but I don't mind.

Reply:
remark added.

Reply:
My initial remark is resolved, so I'm closing this discussion. If extra checks are wishful, please, open a separate discussion thread.